### ISSUE TITLE
Sass Modules - TSLint error when @use and @include mixins with namespaces (in VS Code)

### ISSUE THREAD

**Issue by tomanovy** [palantir/tslint#4901]
### Bug Report / Feature request

- 5.15.0:
- 3.6.4:
- Visual Studio Code  (TSLint Visual Studio Marketplace | v 1.2.3 )

#### TypeScript code being linted

Could someone help how to allow namespaces for linter? I'm not able to google anything relevant for my isue with linter in VS Code...

```scss
// ./my.scss
@use "styles/mix" as mi; 
button {
    @include mi.size('10px');
}
```

with `tslint.json` configuration:

```json
    {
      "extends": "tslint:recommended",
      "rulesDirectory": [
        "codelyzer"
      ],
  "rules": {
    "at-rule-no-unknown": [true, {
      "ignoreAtRules": ["mixin"]
    }],
  // ...
```

#### Actual behavior
after setting "ignoreAtRules": ["mixin"]  there is no more problem reported with "@use" 
(which I was getting with default config)...
..but there are still 2 problems reported:
linter problem 1: mi. <--- semi-colon expected
linter problem 2: ...('10px');}  <--- at-rule or selector expected

#### Expected behavior
@include < namespace >.< mixin > shold not be reported as problem, as it follows specs:
https://sass-lang.com/documentation/at-rules/use 

How to solve this in `tslint.json` configuration, please?
Thanks, Tom

**Comment by JoshuaKGoldberg** [comment]
Hi @tomanovy! It looks like you're looking at `tslint.json`, the configuration file for _TSLint_ - the _TypeScript linter_, and trying to figure out how to apply it to SASS code. SASS code is typically linted by a separate linter such as [stylelint](https://stylelint.io/), perhaps you're using that?

At any rate, if you _do_ need help with rules provided by codelyzer, see https://github.com/mgechev/codelyzer.

...and you might want to take a peek at #4534. TSLint is deprecated. 💀

**Comment by tomanovy** [comment]
Oh, tslint.json is part of default Angular 8 app so I didn't think I should take depreciation urgent... and I totally misunderstood how those linters are configured - now I get it...it's not part of Angular nor VS Code. Thanks for pointing me in right direction and sorry for wasting your time on such bad question.

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
